pennylane/operation.py

```diff
@@ -1036,7 +1036,7 @@ def _obs_data(self):
         tensor = set()
 
         for ob in obs:
-            parameters = tuple(param.tostring() for param in ob.parameters)
+            parameters = tuple(param.tobytes() for param in ob.parameters)
```

Comment:
What's the difference here due to the change?

Reply:
same thing, just one syntax is deprecated.